### ISSUE TITLE
Wide-ResNet 28-10-0.3 for CIAFR10 produced a low test accuracy

### ISSUE THREAD

**Issue by 1017137588qqcom** [meliketoy/Pytorch-wide-resnet#14]
dear Author, 

I downloaded your code and reproduced your experiment on cifar-10 according to your settings, but got 95.24 % test accuracy with cross entropy loss. I am not sure where I have omissions, look forward to your reply！

experimetn settings:
parser.add_argument('--lr', default=0.1, type=float, help='learning_rate')
parser.add_argument('--net_type', default='wide-resnet', type=str, help='model')
parser.add_argument('--depth', default=28, type=int, help='depth of model')
parser.add_argument('--widen_factor', default=10, type=int, help='width of model')
parser.add_argument('--dropout', default=0.3, type=float, help='dropout_rate')

start_epoch = 1
num_epochs = 200
batch_size = 128
optim_type = 'SGD'

mean = {
    'cifar10': (0.4914, 0.4822, 0.4465),
    'cifar100': (0.5071, 0.4867, 0.4408),
}

std = {
    'cifar10': (0.2023, 0.1994, 0.2010),
    'cifar100': (0.2675, 0.2565, 0.2761),
}

optimizer = optim.SGD(net.parameters(), lr=cf.learning_rate(args.lr, epoch), momentum=0.9, weight_decay=5e-4)

the result:
![image](https://user-images.githubusercontent.com/38061052/221138569-e051e7ee-788a-4f59-9aab-928c3e836e6b.png)
